docs/source/API/core/view/resize.rst

```diff
@@ -125,7 +125,7 @@ Description
 * .. code-block:: cpp
 
      template <class T, class... P, class... ViewCtorArgs>
-     void resize(const Impl::ViewCtorProp<ViewCtorArgs...>& arg_prop,
+     void resize(const /* is-alloc-prop */& arg_prop,
 	         Kokkos::View<T, P...>& v,
 	         const typename Kokkos::View<T, P...>::array_layout& layout);
 
```

Comment:
TODO:

- Rebase and see if warnings go away

Reply:
Done. Seeing
```
/home/runner/work/kokkos-core-wiki/kokkos-core-wiki/docs/source/API/containers/DualView.rst:155: WARNING: Error when parsing function declaration.

If the function has no return type:
  Error in declarator or parameters-and-qualifiers
  Invalid C++ declaration: Expected identifier in nested name. [error at 9]
    DualView(/* is-alloc-prop */ const& arg_prop, const size_t n0 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n1 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n2 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n3 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n4 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n5 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n6 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n7 = KOKKOS_IMPL_CTOR_DEFAULT_ARG);
    ---------^
If the function has a return type:
  Error in declarator
  If declarator-id with parameters-and-qualifiers:
    Invalid C++ declaration: Expected identifier in nested name. [error at 8]
      DualView(/* is-alloc-prop */ const& arg_prop, const size_t n0 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n1 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n2 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n3 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n4 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n5 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n6 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n7 = KOKKOS_IMPL_CTOR_DEFAULT_ARG);
      --------^
```
Not sure if this pull request introduced it.

Reply:
i don't think we can have comments inline. in the code : /* is-alloc-prop */

Reply:
Any idea for a workaround?

Reply:
nope :) let me think and get back to you